src/module.js

```diff
@@ -83,7 +83,15 @@
  * });
  * </pre>
  */
-angular.module('stormpath',['stormpath.CONFIG','stormpath.auth','stormpath.userService'])
+angular.module('stormpath',[
```

Comment:
In my files I follow our conventions, but here I just added new dependencies to the list. However, will add the space.